src/test/java/seedu/address/testutil/TypicalPastCommands.java

```diff
@@ -59,19 +59,19 @@ public class TypicalPastCommands {
             createPastCommands(EXAMPLE_CLEAR, EXAMPLE_ADD, EXAMPLE_DELETE, EXAMPLE_DELETEA);
 
 
-    private static CommandHistory CURR_PAST_COMMANDS = new CommandHistory();
+    private static CommandHistory currentPastCommands = new CommandHistory();
```

Comment:
You could convert this field to a local variable, since it is used only in `createPastCommands()`.

Reply:
Have done it this way. Thanks for the suggestion!